openapi/components/schemas/World.yaml

```diff
@@ -9,9 +9,13 @@ properties:
     type: string
   capacity:
     example: 8
-    maximum: 40
```

Comment:
Not sure we should be specifying a min/max for a value we don't control. What is the benefit of this?

Reply:
I agree with not including arbitrary limits, some generators may enforce these while others don't.